alembic/versions/6e1fe3f1eccf_add_merge_requests_comments_reactions_.py

```diff
@@ -0,0 +1,118 @@
+"""Add merge_requests, comments, reactions tables
+
+Revision ID: 6e1fe3f1eccf
+Revises: 70466f47f936
+Create Date: 2025-04-11 09:52:43.840141
+
+"""
+
+from typing import Sequence, Union
+
+import sqlalchemy as sa
+from alembic import op
+
+
+# revision identifiers, used by Alembic.
+revision: str = "6e1fe3f1eccf"
+down_revision: Union[str, None] = "70466f47f936"
+branch_labels: Union[str, Sequence[str], None] = None
+depends_on: Union[str, Sequence[str], None] = None
+
+
+def upgrade() -> None:
+    """Upgrade schema."""
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.create_table(
+        "merge_requests",
+        sa.Column("id", sa.Integer(), nullable=False),
+        sa.Column(
+            "mr_id", sa.Integer(), nullable=False, comment="The merge request id"
+        ),
+        sa.Column("project_id", sa.Integer(), nullable=False, comment="The project id"),
+        sa.Column("job_id", sa.Integer(), nullable=False, comment="The job id"),
+        sa.PrimaryKeyConstraint("id"),
+        sa.UniqueConstraint("mr_id", "project_id", "job_id", name="uix_mr_project_job"),
```

Comment:
just to be clear: when we "rerun" the log detective analysis, will it get a new job id?